api/ocm/extensions/repositories/genericocireg/componentversion.go

```diff
@@ -299,20 +317,74 @@ func (c *ComponentVersionContainer) GetStorageContext() cpi.StorageContext {
 	return ocihdlr.New(c.comp.GetName(), c.Repository(), c.comp.repo.ocirepo.GetSpecification().GetKind(), c.comp.repo.ocirepo, c.comp.namespace, c.manifest)
 }
 
+func blobAccessForChunk(blob blobaccess.BlobAccess, fs vfs.FileSystem, r io.Reader, limit int64) (cpi.BlobAccess, bool, error) {
+	f, err := blobaccess.NewTempFile("", "chunk-*", fs)
```

Comment:
This `f` is never closed in case of a success.

Reply:
I think that's because the file does not have a view. But the blob (`f.AsBlob(...)`) that is returned is based on that file and also closes that file. So you hand the close responsibility over to the blob receiver.